### DISCUSSION TITLE
component bazaar

### DISCUSSION THREAD

**Discussion by WolfgangFahl** [elimintz/justpy#410] [Ideas]
It would be great if [user defined components](https://justpy.io/tutorial/custom_components/) could be shared by the community. A first starting point could be to share ideas for components in github discussions and and a procedure to make them publicly available e.g. in a separate project.

**Comment by WolfgangFahl** [comment]
see #375

**Comment by sandeep-gh** [comment]
Good idea. I have made a hierarchy navigation component using justpy and a
small functional wrapper I wrote on top of justpy. Would be happy to share
:).

**Comment by WolfgangFahl** [comment]
@sandeep-gh  - great - for a start just open a dicussion item and post your source code there. if we get enough such components we might start creating a special webpage for such components or a portal as i did for flask bootstrap features see: http://fb4demo.bitplan.com/. 

I think the current wiki documentation approach in the tutorial could be improved in this respect.

**Comment by sandeep-gh** [comment]
@WolfgangFahl, Need to polish/cleanup somewhat more. Will share soon.

**Comment by WolfgangFahl** [comment]
see http://jpdemo.bitplan.com/ for a first version of an online demo of some of the examples which could be extended to a component bazaar

**Comment by WolfgangFahl** [comment]
see https://github.com/quasarframework/quasar-awesome for quasar examples

**Comment by WolfgangFahl** [comment]
see https://mui.com/

**Comment by WolfgangFahl** [comment]
https://nicegui.io/
